allennlp/nn/util.py

```diff
@@ -835,3 +835,58 @@ def remove_sentence_boundaries(tensor: torch.Tensor,
             new_mask[i, :(j - 2)] = 1
 
     return tensor_without_boundary_tokens, new_mask
+
+
+def add_positional_features(tensor: torch.Tensor,
```

Comment:
@matt-gardner any good ideas for how to test this?

Reply:
Ugh, seems tricky.  Two thoughts:

1. You could compute some values by hand for a couple of different cases, and test that the hand-computed values match what the function produces.
2. Instead of computing the values by hand, compute them using the `tensor2tensor` code, and use those values here in the test, with a note that you got them from the original implementation.

If option 2 is feasible, I'd recommend doing that.